### PR TITLE
Multi platform support for serialport

### PR DESCRIPTION
**Pull request by andrerim**
Adds support for multiple platforms by using the [node-usb-native](https://github.com/VSChina/serialport.node#readme) module maintained by vscode's arduino team. 

This has been tested and works on windows and mac. We used a VM for linux and got the extension to activate and list the serial ports, but could not receive any output or input from the device. Suspect that this is due to issues with accessing a usb port though a VM.

The downside of using this solution, is that we don't get the serialport types.

### CODE REVIEW

File: src/util/serialportLoader.ts [hunk 1]
```diff
@@ -0,0 +1,10 @@
+export class SerialPortLoader {
+    private static _serialport: any; 
+
+    public static get serialport(): any {
+        if (!SerialPortLoader._serialport) {
+            SerialPortLoader._serialport = require("node-usb-native").SerialPort;
+        }
+        return SerialPortLoader._serialport;
+    }
+}
```
Comment (bencefr) [inline src/util/serialportLoader.ts:10]: It is very neat to wrap the require in a class, but consider this:
```
const { SerialPort } = require('node-usb-native');
export default SerialPort;
```

**Review by bencefr — APPROVED** [review]
Looks good to me.

**Comment by michaeljones** [comment]
We've merged this but it seems that the published node-usb-native package does not yet support Electron 12 which the very latest VSCode is based on. We're not sure when they might republish. There isn't a much in the way of communication on that repository (https://github.com/VSChina/serialport.node)

**Comment by michaeljones** [comment]
This fork seems to be trying to build it for Electron 12 at the moment: https://github.com/curdeveryday/serialport.node/commits/master. I can't tell if that person is part of the Microsoft team and might be able to contribute the builds back to the main package or not. I guess we'll see in a few days.

**Pull request merged**